doc/news/changes/minor/20171020GiovanniAlzetta

```diff
@@ -0,0 +1,4 @@
+New: Added a method and a enum class to the class BoundingBox. The method is get_neighbor_type which, given another BoundingBox, returns the enum type NeighborType. This enum class describes how the bounding boxes are neighbor.
```

Comment:
Please shorten and include the fully qualified name of the class. Maybe something like
"Added BoundingBox::get_neighbor_type which [...]"